src/path_handler.cpp

```diff
@@ -106,7 +106,7 @@ double PathHandler::getMaxCostmapDist()
 }
 
 nav_msgs::msg::Path PathHandler::transformPlanPosesToCostmapFrame(
-  PathIterator begin, PathIterator end, const StampType & stamp)
+  PathIterator & begin, PathIterator & end, const StampType & stamp)
```

Comment:
better pass by value

Reply:
I think in best case it shouldn't matter, but in worst case it would. I'd think defensively we'd just want to keep this one

Reply:
I've just never saw passing iterators by reference. I'm almost sure it's like pointer in size

Reply:
fair enough